src/amo/components/SearchFilters/index.js

```diff
@@ -128,6 +128,9 @@ export class SearchFiltersBase extends React.Component<InternalProps> {
       delete filters.category;
       delete filters.addonType;
     }
+    if (filters.tag) {
```

Comment:
Please add a comment similar to the one for category. This isn't something we normally do (deleting filters).